mobly/snippet/client_base.py

```diff
@@ -107,14 +97,13 @@ def __init__(self, package, device):
   def __del__(self):
     self.close_connection()
 
-  def start_server(self):
-    """Starts the server on the remote device and connects to it.
+  def initialize(self):
+    """Initializes the snippet client to interact with the remote device.
 
-    This process contains four stages:
-      - before_starting_server: prepares for starting the server.
-      - do_start_server: starts the server on the remote device.
-      - build_connection: builds a connection with the server.
-      - after_starting_server: does the things after the server is available.
+    This function contains following stages:
+      - preparing for starting the snippet server.
```

Comment:
use numbered bullet points to emphasize that these are ordered?

Reply:
Done.